src/services/dataService/user.js

```diff
@@ -0,0 +1,41 @@
+const {User} = require('./db')
+
+function findUsers() {
+	return User.findAll()
+}
+
+function getUser(id) {
+  return User.findOne({
+    where: {id},
+  })
+}
+
+function createUser(name, email, passwordHash) {
+	return User.create({
+		name: name,
+		email: email,
+		passwordHash: passwordHash
+	})
+}
+
+function updateUser(id) {
+	return User.findOne({
```

Comment:
Need to accept also the values to be updated and pass them in. Also need to be using the `User.update` method here (instead of `findOne`).

Example:
```
function updateUser(id, data) {
  return User.update(data, {where: {id}, returning: true})
    .then(updates => {
      const [numUpdated, updatedUsers] = updates
      return updatedUsers[0]
    })
}
```